mock-server/app.js

```diff
@@ -3,31 +3,19 @@ const express = require('express');
 const logger = require('morgan');
 const cors = require('cors');
 const plantronics = require('./plantronics/router');
-// const sennheiser = require('./sennheiser').router;
+const sennheiser = require('./sennheiser').router;
```

Comment:
NIT: It feels slightly odd to have `plantronics` export the router via a single file but then have `sennheiser` export via a barrel file.